tfx/components/example_gen/base_example_gen_executor_test.py

```diff
@@ -142,6 +143,41 @@ def testDoOutputSplit(self):
     # Check example gen outputs.
     self.assertTrue(tf.io.gfile.exists(self._train_output_file))
     self.assertTrue(tf.io.gfile.exists(self._eval_output_file))
+    
+    # Output split ratio: train:eval=2:1.
+    self.assertGreater(
+        tf.io.gfile.GFile(self._train_output_file).size(),
+        tf.io.gfile.GFile(self._eval_output_file).size())
+    
```

Comment:
please add tests for different invalid cases, 
1. used float_list feature
2. config a feature that does not exist in some examples